--- VTK-6.1.0/Rendering/FreeType/vtkFreeTypeTools.cxx.orig	2015-02-18 19:39:39.428358188 +0100
+++ VTK-6.1.0/Rendering/FreeType/vtkFreeTypeTools.cxx	2015-02-18 19:40:48.241691988 +0100
@@ -1186,7 +1186,7 @@
     if (bitmap)
       {
       metaData.ascent = std::max(bitmapGlyph->top - 1, metaData.ascent);
-      metaData.descent = std::min(-(bitmap->rows - (bitmapGlyph->top - 1)),
+      metaData.descent = std::min(-((int)bitmap->rows - ((int)bitmapGlyph->top - 1)),
                                   metaData.descent);
       }
     ++heightString;
@@ -1951,8 +1951,8 @@
     if (bitmap)
       {
       bbox[0] = std::min(bbox[0], pen[0] + bitmapGlyph->left);
-      bbox[1] = std::max(bbox[1], pen[0] + bitmapGlyph->left + bitmap->width);
-      bbox[2] = std::min(bbox[2], pen[1] + bitmapGlyph->top - 1 - bitmap->rows);
+      bbox[1] = std::max(bbox[1], pen[0] + bitmapGlyph->left + (int)bitmap->width);
+      bbox[2] = std::min(bbox[2], pen[1] + bitmapGlyph->top - 1 - (int)bitmap->rows);
       bbox[3] = std::max(bbox[3], pen[1] + bitmapGlyph->top - 1);
       }
     else
