--- VTK-8.2.0/IO/Movie/CMakeLists.txt.orig	2019-01-30 18:15:13.000000000 +0100
+++ VTK-8.2.0/IO/Movie/CMakeLists.txt	2020-08-02 01:01:33.766555218 +0200
@@ -29,6 +29,7 @@
   ${CMAKE_CURRENT_BINARY_DIR}/vtkIOMovieConfigure.h)
 
 vtk_module_library(vtkIOMovie ${Module_SRCS})
+vtk_module_link_libraries(vtkIOMovie LINK_PUBLIC ${THEORA_dec_LIBRARY} ${THEORA_enc_LIBRARY} ${OGG_LIBRARY})
 
 if(WIN32 AND VTK_USE_VIDEO_FOR_WINDOWS)
   vtk_module_link_libraries(vtkIOMovie LINK_PRIVATE vfw32)
--- VTK-8.2.0/ThirdParty/exodusII/vtkexodusII/CMakeLists.txt~	2019-01-30 18:15:13.000000000 +0100
+++ VTK-8.2.0/ThirdParty/exodusII/vtkexodusII/CMakeLists.txt	2020-08-02 01:13:25.706301562 +0200
@@ -299,6 +299,7 @@
 
 vtk_add_library(vtkexodusII ${sources} ${headers})
 target_link_libraries(vtkexodusII PUBLIC ${vtknetcdf_LIBRARIES})
+target_link_libraries(vtkexodusII PUBLIC ${CMAKE_THREAD_LIBS})
 if (NOT VTK_INSTALL_NO_DEVELOPMENT)
   install(FILES
     ${headers}
--- VTK-8.2.0/ThirdParty/libharu/vtklibharu/CMakeLists.txt.orig	2020-08-02 10:14:54.000000000 +0200
+++ VTK-8.2.0/ThirdParty/libharu/vtklibharu/CMakeLists.txt	2020-08-02 10:25:09.955242344 +0200
@@ -298,3 +297,5 @@
 
 INCLUDE( CPack )
 endif ()
+
+find_library(M_LIBRARY m)
--- VTK-8.2.0/ThirdParty/libharu/vtklibharu/src/CMakeLists.txt~	2019-01-30 18:15:13.000000000 +0100
+++ VTK-8.2.0/ThirdParty/libharu/vtklibharu/src/CMakeLists.txt	2020-08-02 10:25:38.091690113 +0200
@@ -113,3 +113,4 @@
       "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>"
       "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/../include>")
 endif ()
+target_link_libraries(vtklibharu PUBLIC ${M_LIBRARY})
