--- VTK-8.2.0/Rendering/Qt/vtkQtLabelRenderStrategy.cxx~	2019-01-30 18:15:13.000000000 +0100
+++ VTK-8.2.0/Rendering/Qt/vtkQtLabelRenderStrategy.cxx	2020-08-01 18:48:26.642080520 +0200
@@ -41,6 +41,7 @@
 #include <QImage>
 #include <QMap>
 #include <QPainter>
+#include <QPainterPath>
 #include <QPair>
 #include <QPixmap>
 #include <QTextDocument>
--- VTK-8.2.0/Rendering/Qt/vtkQtStringToImage.cxx~	2019-01-30 18:15:13.000000000 +0100
+++ VTK-8.2.0/Rendering/Qt/vtkQtStringToImage.cxx	2020-08-01 18:57:14.025679183 +0200
@@ -31,6 +31,7 @@
 #include <QFontMetrics>
 #include <QImage>
 #include <QPainter>
+#include <QPainterPath>
 #include <QPixmap>
 #include <QTextDocument>
 #include <QTextStream>
