--- VTK-8.2.0/ThirdParty/pugixml/CMakeLists.txt.orig	2020-08-07 00:25:58.603999094 +0200
+++ VTK-8.2.0/ThirdParty/pugixml/CMakeLists.txt	2020-08-07 00:26:12.843915244 +0200
@@ -1,10 +1 @@
-vtk_module_third_party(pugixml
-  LIBRARIES vtkpugixml
-  INCLUDE_DIRS ${vtkpugixml_SOURCE_DIR}/vtkpugixml
-  ${vtkpugixml_BINARY_DIR}/vtkpugixml
-  NO_ADD_SUBDIRECTORY)
-
-vtk_module_export_info()
-add_subdirectory(vtkpugixml)
-
-vtk_target_export(vtkpugixml)
+vtk_module_third_party(pugixml)
