--- VTK-8.2.0/IO/CityGML/CMakeLists.txt.orig	2019-01-30 18:15:13.000000000 +0100
+++ VTK-8.2.0/IO/CityGML/CMakeLists.txt	2021-01-05 09:43:26.611074217 +0100
@@ -5,7 +5,7 @@
 if(VTK_USE_SYSTEM_PUGIXML AND NOT pugixml_INCLUDE_DIRS)
   find_package(pugixml REQUIRED)
-  get_target_property(pugixml_INCLUDE_DIRS pugixml INTERFACE_INCLUDE_DIRECTORIES)
-  get_target_property(pugixml_LIBRARIES pugixml LOCATION)
+  get_target_property(pugixml_INCLUDE_DIRS pugixml::pugixml INTERFACE_INCLUDE_DIRECTORIES)
+  get_target_property(pugixml_LIBRARIES pugixml::pugixml INTERFACE_LINK_LIBRARIES)
   include_directories(${pugixml_INCLUDE_DIRS})
   vtk_module_link_libraries(vtkIOCityGML LINK_PRIVATE ${pugixml_LIBRARIES})
 endif()
