diff -up VTK-7.1.0/Wrapping/Tools/vtkParsePreprocess.c.abi VTK-7.1.0/Wrapping/Tools/vtkParsePreprocess.c
--- VTK-7.1.0/Wrapping/Tools/vtkParsePreprocess.c.abi	2016-11-14 09:59:47.000000000 -0700
+++ VTK-7.1.0/Wrapping/Tools/vtkParsePreprocess.c	2016-12-01 16:12:45.793113151 -0700
@@ -2445,6 +2445,21 @@ void vtkParsePreprocess_AddStandardMacro
 #ifdef __LITTLE_ENDIAN__
     PREPROC_ADD_MACRO(info, __LITTLE_ENDIAN__);
 #endif
+#ifdef __CHAR_UNSIGNED__
+    PREPROC_ADD_MACRO(info, __CHAR_UNSIGNED__);
+#endif
+#ifdef __SIZEOF_POINTER__
+    PREPROC_ADD_MACRO(info, __SIZEOF_POINTER__);
+#endif
+#ifdef _LP64
+    PREPROC_ADD_MACRO(info, _LP64);
+#endif
+#ifdef _ILP32
+    PREPROC_ADD_MACRO(info, _ILP32);
+#endif
+#ifdef __64BIT__
+    PREPROC_ADD_MACRO(info, __64BIT__);
+#endif
   }
 
   info->IsExternal = save_external;
